rmgpy/rmg/model.py

```diff
@@ -1135,8 +1135,7 @@ def add_species_to_core(self, spec):
 
         rxn_list = []
         if spec in self.edge.species:
-            if not self.edge.phase_system.in_nose:
-                self.edge.phase_system.pass_species(spec.label,self.core.phase_system)
+            self.edge.phase_system.pass_species(spec.label,self.core.phase_system)
```

Comment:
Very happy that now we can run these tests on RMS reactors

Reply:
:tada: I am planning a follow up PR to add unit tests to that effect